build.Dockerfile

```diff
@@ -123,6 +123,7 @@ WORKDIR /usr/local/bin
 
 COPY --from=builder /opt/sgxsdk/lib64 /opt/sgxsdk/lib64
 COPY --from=builder /root/work/worker/bin/* ./
+COPY --from=builder /lib/x86_64-linux-gnu /lib/x86_64-linux-gnu
```

Comment:
This copies some additional DCAP libraries, but also many libraries that we don't need. It makes our images quite a bit bigger.
Something I still want to understand and discuss with Felix.